### PR TITLE
Create ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by rhyshort**
Create initial ISSUE_TEMPLATE.md to help users supply us with the information needed 
to quickly reproduce issues.

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Please include the following information in your ticket.
+
+- Version(s) of CloudantClient which this issue applies
```
Comment (ricellis) [inline ISSUE_TEMPLATE.md]: CloudantClient (java-cloudant) version(s) that are affected by this issue.

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+- Java Version include vendor and platform.
```
Comment (ricellis) [inline ISSUE_TEMPLATE.md]: Java version (including vendor and platform).

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+- If you're using the optional okhttp dependency
```
Comment (ricellis) [inline ISSUE_TEMPLATE.md]: Needs a `.` at the end.

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (rhyshort) [inline ISSUE_TEMPLATE.md]: done in 8f3de7e

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (rhyshort) [inline ISSUE_TEMPLATE.md]: done in 59fe3a8

File: ISSUE_TEMPLATE.md (hunk 3 above)
Comment (rhyshort) [inline ISSUE_TEMPLATE.md]: done in 434c955

**Comment by ricellis** [comment]
+1

**Pull request merged**